packages/create-sitecore-jss/src/templates/angular/src/app/components/styleguide-section/styleguide-section.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="pt-3" [id]="id">
+  <h3 class="border-bottom" *scText="rendering.fields.heading"></h3>
+  <sc-placeholder name="<%- appPrefix ? `${helper.getPascalCaseName(appName)}-` : "" %>-jss-styleguide-section" [rendering]="rendering"></sc-placeholder>
```

Comment:
If `appPrefix` is empty, here you will have `-jss-styleguide-section`, remove `-`